pyproject.toml

```diff
@@ -25,7 +25,7 @@ python-dateutil = "^2.8.2"
 requests = "^2.25.0"
 cryptography = ">=3.4.8"
 tqdm = "^4.32.1"
-pydantic = ">=1.9.0"
+pydantic = ">=1.10.14"
```

Comment:
Can we avoid bumping pydantic?

Reply:
Not without mucking around with the API payloads; Pydantic (or, really, the `typing_extensions`) had a bug which breaks any models that have `Literal` fields.